src/components/redemption/Confirming.js

```diff
@@ -24,12 +24,10 @@ const Confirming = ({ txHash, btcNetwork, error }) => {
           <p>We're waiting to confirm your transaction.</p>
           {
             txHash
-            ? <button className="black">
-                <a href={blockExplorerUrl}
+            ? <a className=""href={blockExplorerUrl}
```

Comment:
May want to add the space before href here.

Reply:
And remove the `className`!

Reply:
Removed in 766004b